fireapp/lib/presentation/calendar/calendar_widget.dart

```diff
@@ -188,7 +188,7 @@ class _CalendarState extends FireAppState<CalendarView>
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
             Text(displayTime),
-            Text('Status: ${shiftEvent.status}'),
+            Text('${AppLocalizations.of(context)?.status}: ${shiftEvent.status}'),
```

Comment:
This isn't how you do it. Look at `formFieldEmpty` for the correct method.

Reply:
We can't just interpolate strings like this because: 
1. It would end up being messy if, for example, the interpolation was in the middle. You would need to do something like `'${AppLocalizations.of(context}?.start} ${middle} ${AppLocalizations.of(context}?.end}'`
2. What if another language would expect the interpolated content to be elsewhere? We would be unable to translate the app correctly